services/account/k6-test/README.md

```diff
@@ -22,5 +22,11 @@ To run the script, you need to have k6 installed. You can download it from [here
 To run the script, execute the following command:
 
 ```bash
-k6 run health-check.js
+k6 run health-check.k6.js
 ```
+
+## Generating Test Data
```

Comment:
Added a script to generate the information needed to run this test. I went for committing the generated code, as it works fine for running in CI and most runs. I'd be open to generating this as needed as well.